deployment/internal/soltestutils/preload.go

```diff
@@ -12,7 +18,14 @@ import (
 func LoadKeystonePrograms(t *testing.T, dir string) map[string]string {
 	t.Helper()
 
-	return directory.LoadKeystonePrograms(t, dir)
+	keystoneProgIDs := loadProgramArtifacts(t, solutils.KeystoneProgramNames, downloadChainlinkSolanaProgramArtifacts, dir)
```

Comment:
Removed a level of indirection and performs the loading here instead.

This is the same as below